archiver.rb

```diff
@@ -0,0 +1,32 @@
+require "tempfile"
```

Comment:
All of these modules are not needed, the File class is part of the Ruby Core, if for some reason you would need to use some of the methods of those modules then it is needed to require them, but at this point, you are not taking advantage of them
https://ruby-doc.org/core-2.5.0/File.html